src/components/pages/product-page/ProductPage.tsx

```diff
@@ -2,15 +2,12 @@ import React from 'react';
 import { ComponentMapper } from '../../ComponentMapper';
 import { ProductPageProps } from '../../../types/content-props/dynamic-page-props';
 import { ThemedPageHeader } from '../../_common/headers/themed-page-header/ThemedPageHeader';
-import { BEM } from '../../../utils/classnames';
-
-const bem = BEM('product-page');
 
 export const ProductPage = (props: ProductPageProps) => {
     return (
-        <div className={bem()}>
+        <div className={'productPage'}>
             <ThemedPageHeader contentProps={props} />
-            <div className={bem('content')}>
+            <div className={'content'}>
```

Comment:
trenger vi ikke fortsatt BEM her for å skille fra content i SituationPage.scss?

Reply:
Nei, fordi den ligger hierarikisk under .productPage